source/reference/mongosync/mongosync-behavior.txt

```diff
@@ -133,6 +133,13 @@ Read Preference
 connecting to the source and destination clusters. For more information,
 see :ref:`connections-read-preference`. 
 
+Rewrites
+~~~~~~~~
+
+``mongosync`` rewrites any legacy index values from the source to ``1``
```

Comment:
Perhaps, 

"``mongosync`` does not support legacy index values, like ``0`` or an empty
string. Legacy index values from the source are rewritten to ``1``
on the destination."